gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/api/domain_service/VerifyApiPathDomainService.java

```diff
@@ -20,5 +20,5 @@
 import java.util.List;
 
 public interface VerifyApiPathDomainService {
-    List<Path> verifyApiPaths(ExecutionContext executionContext, String apiId, List<Path> paths);
+    List<Path> verifyApiPaths(String environmentId, String apiId, List<Path> paths);
```

Comment:
It seems that the method does more than "verify" but also sanitized and could change paths value. I would rename the method to be more explicit about it actually does.

Reply:
I wanted to avoid a too long name but you're right it verifies the paths, and returns a list of corresponding sanitized paths. Any name suggestion ?

Reply:
why not `checkAndSanitizePaths` ?